portfolio/src/main/webapp/assets/fullpage/fullpage.css

```diff
@@ -0,0 +1,236 @@
+/*!
+ * fullPage 3.0.8
+ * https://github.com/alvarotrigo/fullPage.js
+ *
+ * @license GPLv3 for open source use only
+ * or Fullpage Commercial License for commercial use
+ * http://alvarotrigo.com/fullPage/pricing/
+ *
+ * Copyright (C) 2018 http://alvarotrigo.com/fullPage - A project by Alvaro Trigo
+ */
```

Comment:
It doesn't matter too much with a personal toy projects. For more serious projects, keep pulled in (vendored) code in a dedicated directory (e.g. //third-party) is helpful to manage their version, license, etc.